lib/rspec/core/example_group.rb

```diff
@@ -111,16 +111,14 @@ def described_class
       #   @overload $1
       #   @overload $1(&example_implementation)
       #     @param example_implementation [Block] The implementation of the example.
-      #   @overload $1(doc_string, *metadata_keys, metadata={})
```

Comment:
Why are you removing this? Its the same that you're adding below 😕

Reply:
I don't have a strong preference for how to do it.
What is the best way to give the idea that it supports several keys and also a hash from your point of view?

Reply:
Can you generate the docs so we can compare them with current yard?

Reply:
Here you go https://github.com/rspec/rspec.github.io/pull/132/commits/d7ec1cdf43ba06dc2f65c2d28d88258b20f4b2f1
[Full update](https://github.com/rspec/rspec.github.io/pull/132) - this is hard to make considering permissive yard versioning across repos, fixed it temporarily to 0.9.16.